app/client/components/payment/update/updatePaymentFlow.tsx

```diff
@@ -128,32 +129,71 @@ const PaymentMethodRadioButton = (props: PaymentMethodRadioButtonProps) => {
     }
   `;
 
-  return (
+  const label = () => (
```

Comment:
The following changes is due to radio buttons not being fully clickable. e.g.: hovering the mouse over the card logos would prevent the radio from being clicked.